docs/custom-components/custom-components-command-reference.md

```diff
@@ -0,0 +1,202 @@
+# Custom Components Command Reference
```

Comment:
This page can just be called `api-reference.md` or `commands.md` since the `custom-components` is already part of the URL